recipes/cxxopts/all/test_package/conanfile.py

```diff
@@ -26,11 +26,11 @@ def test(self):
             output = StringIO()
             bin_path = os.path.join(self.cpp.build.bindirs[0], "test_package")
             option_string = "-f 41 --bar baria --baz"
-            if self.options["cxxopts"].unicode:
+            if self.dependencies["cxxopts"].options.unicode:
```

Comment:
it can't work AFAIK

Reply:
indeed! forgot I was in the `test()` method. Will fix!

Reply:
Maybe use ctest trick you've proposed in boost recipe?

Reply:
https://github.com/conan-io/conan-center-index/pull/15091#issuecomment-1372798369 🙏 

This looks super promising from Space!

Reply:
I am stealing https://github.com/conan-io/conan-center-index/pull/15725 :)